### DISCUSSION TITLE
How Expected Improvement is calculated for multi objective functions

### DISCUSSION THREAD

**Discussion by lovecodng** [meta-pytorch/botorch#817] [Q&A] [answered]
Can you please explains how the expected improvement  is calculated for multi-objective functions I mean the equation from the q`ExpectedImprovement` function is `qEI(X) = E(max(max Y - best_f, 0))`, its confusing how you can decide on the max or best values in the presence of the trade off between the different objectives.

**Comment by Balandat** [comment, marked as answer]
EI doesn't directly extend to the multi-objective case, something that's somewhat of an analogue is expected hypervolume improvement. You'll want to take our look at our recent papers on the topic: 
https://papers.nips.cc/paper/2020/hash/6fec24eac8f18ed793f5eaad3dd7977c-Abstract.html
https://arxiv.org/abs/2105.08195

**Reply by lovecodng** [reply]
Thank you

**Reply by lovecodng** [reply]
@Balandat I am reading one of the papers now, but I have a question do you covert the MO to SO objective when optimizing the Acquisition function

**Reply by Balandat** [reply]
I am not sure I understand. EHVI as described in the papers is a multi-objective acquisition function, but it itself is scalar-valued. We optimize that. There are other alternatives that we discuss in the papers.
